inst/shiny/ui.R

```diff
@@ -21,6 +21,7 @@ library(base)
 library(SingleCellExperiment)
 library(singleCellTK)
 library(shinycssloaders)
+library(shinythemes)
```

Comment:
@mmkhan19 All packages that you want to import have to be listed in the description